lib/ansible/modules/web_infrastructure/jenkins_plugin.py

```diff
@@ -340,6 +340,7 @@ def _get_json_data(self, url, what, **kwargs):
             self.module.fail_json(
                 msg="Cannot parse %s JSON data." % what,
                 details=to_native(e))
+            raise e
```

Comment:
Please correct me if I'm wrong, but this `raise e` is never evaluated because the execution exists on the `fail_json()` call as visible [here](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/basic.py#L2284).